frontend/package.json

```diff
@@ -10,9 +10,15 @@
     "format": "prettier --write src"
   },
   "dependencies": {
+    "@codemirror/autocomplete": "^6.9.1",
+    "@codemirror/lang-markdown": "^6.2.1",
```

Comment:
Replace MD with Java or JS, up to you

Reply:
replaced with js, added eslint as linter